### PR TITLE
Add screenshots of what ordering a Let's Encrypt cert looks like

### LINKED ISSUE #222: Add screenshots to the Let's Encrypt order article and limitations article
Add UI screenshots to the Let's Encrypt order article (to show and explain the various fields) and limitations article (to show the different limits based on plans).

### PR DESCRIPTION
**Pull request by jcaudle**
This will close #222

### CODE REVIEW

File: content/articles/ordering-lets-encrypt-certificate.markdown [hunk 1]
```diff
@@ -58,6 +58,20 @@ The order is the first step into getting an SSL certificate. It will create an S
     1.  Submit the order.
 </div>
 
+### What this looks like
+
+As mentioned above, you will see a different form for ordering a Let's Encrypt certificate based on the plan to which you are subscribed. If you have full access to SAN certificate configuration, this is what you'll see:
+
+![Let's Encrypt with SAN](/files/lets-encrypt-new-san.png)
+
+On a less advanced plans you will see the ability to provide only one name on your Let's Encrypt certificate:
```
Comment (weppos) [inline content/articles/ordering-lets-encrypt-certificate.markdown]: > less advanced plans

I actually don't really like this classification. I'd simply use Personal/Professional/Business for now.

Comment (weppos) [inline content/articles/ordering-lets-encrypt-certificate.markdown]: Or you can use:

> If your plan allows A and B, then
> If your plan allows A, then
> Else

And at the end you can leave a <note> where you explain which plans have what.

Comment (jcaudle) [inline content/articles/ordering-lets-encrypt-certificate.markdown]: I think I've addressed this concern in ee4eb6d.

**Comment by jcaudle** [comment]
@weppos I didn't note the plans that have which features as I'm not completely sure which ones have what.

**Comment by weppos** [comment]
I'm fine for now. We can add them later after the public beta.

**Pull request merged**